doc/source/whatsnew/v0.23.0.txt

```diff
@@ -476,7 +476,11 @@ MultiIndex
 - Bug in :func:`MultiIndex.get_level_values` which would return an invalid index on level of ints with missing values (:issue:`17924`)
 - Bug in :func:`MultiIndex.remove_unused_levels` which would fill nan values (:issue:`18417`)
 - Bug in :func:`MultiIndex.from_tuples`` which would fail to take zipped tuples in python3 (:issue:`18434`)
--
+- Bug in :func:`MultiIndex.get_loc`` which would fail to automatically cast values between float and int (:issue:`18818`, :issue:`15994`)
```

Comment:
i count 5 issues here, but 6 in the top of the PR?

Reply:
[Already](https://github.com/pandas-dev/pandas/pull/19074#discussion_r161966691) replied [twice](https://github.com/pandas-dev/pandas/pull/19074#discussion_r161219578)

Reply:
and I still want it listed appropriately.

Reply:
OK, but it's not a bug, where shall I list it.